.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,65 @@
+name: Publish Site
+
+
+on:
+  workflow_dispatch:
```

Comment:
Could we deploy on each push of a tag (=release), to not have to think about it?

Reply:
Good idea, I'll add it, although I was expecting you to convert it to workflows-kt :)

(I copied this workflow from another project, and it was triggered by another workflow.)

Reply:
Ok, I can convert it in a follow-up PR :)